### DISCUSSION TITLE
Regarding the Issue of Obtaining Azure Benefits through GitHub Student Verification.

### DISCUSSION THREAD

**Discussion by WaterColor201** [community/community#138386] [GitHub Education]
### Hi everyone,

I used my educational email to verify my student status on GitHub during my undergraduate studies and took advantage of Azure's student benefits. Now that I have advanced to a higher level of education and have a new educational email from the same school, I encountered the issue shown in the image when trying to verify my student status on GitHub again. As a result, I cannot use my current email for Azure benefits. What should I do? Thank you.
![15D1A1D098954A36C8A53AB82F2D6D5F](https://github.com/user-attachments/assets/b6a45297-e455-4b14-91ed-7152186f711d)

**Comment by itisrude** [comment]
Hi WaterColor201,

GitHub has probably already verified you as a student, so for now, you don't have to. If you're trying to switch over to your new email, here are a couple of things you could try:

- Check to see if the old student status is still listed active under the GitHub Student Developer Pack area in your account settings

- If that doesn't work, just reach out to GitHub support, and they can help sort this out for you.

You might also be able to continue using your old email for now, as long as it is still affiliated with the same school.

Hope this helps!

**Reply by WaterColor201** [reply]
Thank you for your reply!Before I submmit the discussion,I have already delete my old edu email.But I still can't authenticate my new email in github.So it seems that the only way is to reach the Github support.Thank you again for your reply!

**Reply by itisrude** [reply]
Hi WaterColor201,

Ah, got it! Since the old email has already been deleted and you’re still unable to verify the new one, yeah, reaching out to GitHub support is probably your best option at this point. They should be able to help update your account with the new email and fix the issue.

Hope they get it sorted out for you soon!

**Reply by WaterColor201** [reply]
I'll try it,thank you very much!

**Reply by itisrude** [reply]
No worries! If this helped solve your issue, feel free to mark it as solved.
